src/test/java/com/artipie/npm/NpmITCase.java

```diff
@@ -153,6 +166,46 @@ void npmPublish() throws Exception {
         );
     }
 
+    @Test
+    void npmInstallFailWithForbidden() throws Exception {
+        final ArtipieServer.User user = ArtipieServer.BOB;
+        this.init(false);
+        this.saveFilesToStrg(NpmITCase.PROJ);
+        MatcherAssert.assertThat(
+            this.exec(
+                "npm", "install", NpmITCase.PROJ,
+                "--registry", this.url(Optional.of(user))
+            ).getStderr().replaceAll("\n", ""),
+            new StringContains(
+                String.format(
+                    "npm ERR! 403 403 Forbidden - GET http://%s:***@%s:%d/my-npm/%s",
+                    user.name(), NpmITCase.HOST, this.port, NpmITCase.PROJ.replace("/", "%2f")
+                )
```

Comment:
@genryxy it looks like here you check too many details about output. I think it will be enough to see that there is error message with `403 Forbidden` text. Building whole URL looks excessive.